From 37868aebe6d4e92059ffaebe54fe69b7c436ef79 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:22:22 -0400
Subject: [redhat] kabi: Adding symbol __vmalloc (mm/nommu.c)

Message-id: <e6dc8191db3d5b61f5750f1dfc6013c590bd57b6.1410510773.git.poros@redhat.com>
Patchwork-id: 95919
O-Subject: [PATCH RHEL7.1 022/233] kabi: Adding symbol __vmalloc (mm/nommu.c)
Bugzilla: 1062361 1071097
RH-Acked-by: Larry Woodman <lwoodman@redhat.com>

Bugzilla: 1062361,1071097
    https://bugzilla.redhat.com/show_bug.cgi?id=1062361
    https://bugzilla.redhat.com/show_bug.cgi?id=1071097

    Description:
    ============
    Used to large virtually contiguous memory with gfp flags, as vmalloc does not take gfp flags.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/__vmalloc
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/__vmalloc
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/__vmalloc

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/__vmalloc  b/redhat/kabi/kabi-module/kabi_ppc64/__vmalloc 
new file mode 100644
index 0000000..99cd14c
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/__vmalloc  
@@ -0,0 +1,2 @@
+#1-
+0x26477c07 __vmalloc vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/__vmalloc  b/redhat/kabi/kabi-module/kabi_s390x/__vmalloc 
new file mode 100644
index 0000000..00ee03f
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/__vmalloc  
@@ -0,0 +1,2 @@
+#1-
+0x8e879bb7 __vmalloc vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/__vmalloc  b/redhat/kabi/kabi-module/kabi_x86_64/__vmalloc 
new file mode 100644
index 0000000..2e1ade5
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/__vmalloc  
@@ -0,0 +1,2 @@
+#1-
+0xa9bd2676 __vmalloc vmlinux EXPORT_SYMBOL
-- 
1.7.1